controllers/spacerequest/spacerequest_controller_test.go

```diff
@@ -310,6 +310,60 @@ func TestCreateSpaceRequest(t *testing.T) {
 				HasTier(sr.Spec.TierName).
 				HasParentSpace("jane") // the parent space is set as expected
 		})
+
+		t.Run("subSpace target cluster is different from spacerequest cluster", func(t *testing.T) {
+			// given
+			spaceRequest := spacerequesttest.NewSpaceRequest("jane", "jane-tenant",
+				spacerequesttest.WithTierName("appstudio"),
+				spacerequesttest.WithTargetClusterRoles([]string{"member-2"})) // the provisioned namespace is targeted for member-2
```

Comment:
How do I know which cluster the parent-space is provisioned in? 
Looking at the declaration of `parentSpace`:
```
parentSpace := spacetest.NewSpace(test.HostOperatorNs, "jane")
```
it doesn't specify the member cluster.
Shouldn't we do:
```
parentSpaceWithTarget := spacetest.NewSpace(test.HostOperatorNs, "jane", spacetest.WithSpecTargetCluster("member-2"))
```

Reply:
You are right, thanks for the suggestion! 

Done in https://github.com/codeready-toolchain/host-operator/pull/841/commits/1e7cdc3276481f3527f3510e63cedb61637e7088